config/rollup.config.aliases.js

```diff
@@ -0,0 +1,12 @@
+import memory from 'rollup-plugin-memory';
+import rollupConfig from './rollup.config';
+
+export default Object.assign({}, rollupConfig, {
+	plugins: [
+		memory({
+			path: 'src/preact',
+			contents: `import { h } from './preact';export * from './preact';export { h as createElement };`
```

Comment:
Wondering if it's worth pulling this into a file (`src/aliases.js`?) for visibility.  Or if there's anything else we might add in the future, it'd be easier in a file. Thoughts?